Sources/SmileID/Classes/Networking/SmileIDService.swift

```diff
@@ -192,6 +219,54 @@ public class SmileIDService: SmileIDServiceable, ServiceRunnable {
         post(to: "upload", with: request)
     }
 
+    public func doSmartSelfieEnrollment(
+        signature: String,
+        timestamp: String,
+        selfieImage: MultipartBody,
+        livenessImages: [MultipartBody],
```

Comment:
Can we make this `Data` and `[Data]`, and do the conversion to the MultipartBody in this method body? That way the callers don't need to work about this detail